index.html

```diff
@@ -47,160 +57,211 @@ <h1 class="introduction-title">Hey there, I’m Matama <span class="title-green"
-        <li><a href="#" class="anc card-a4">html</a></li>
-      </ul>
-      <a href="#" class="tag">
-        <button type="button" class="btn btn-success see-button">See Project</button>
-      </a>
-    </div>
+      <div class="card">
+        <div class="card-image car"></div>
+        <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+          <li><a href="#" class="anc card-a2">CSS</a></li>
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
 
- <!-- About me -->
-    <div class="about-me">
-      <h1 class="intro">About Me</h1>
-      <p class="about-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-don’t hestiate to contact me.</p>
-      <img src="images/about.svg" alt="">
-      <div class="text-center">
+      <div class="card">
+        <div class="card-image car"></div>
+        <h2 class="card-text">Multi-Post Stories Gain+Glory</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc card-a1">Rubyonrails</a></li>
+          <li><a href="#" class="anc card-a2">CSS</a></li>
+          <li><a href="#" class="anc card-a3">Javscript</a></li>
+          <li><a href="#" class="anc card-a4">html</a></li>
+        </ul>
+        <a href="#" class="tag">
+          <button type="button" class="btn btn-success see-button">See Project</button>
+        </a>
+      </div>
+      </div>
+
+      <!-- About me -->
+      <div class="about-me">
+        <div class="about-me-text">
+        <h1 class="intro">About Me</h1>
+        <p class="about-text"> I can help you build a product , feature or website Look through some of my work and
+          experience! If you like what you see and have a project you need coded,
+          don’t hestiate to contact me.</p>
+        </div>
+        <div class="text-center">
           <button type="button" class="btn btn-success resume-button">Get My Resume</button>
+        </div>
       </div>
-    </div>
 
-    <div class="card">
-      <div class="card-image sphere">
-        <img src="images/sphere.svg" class="sp-image" alt="">
+      <div class="work-cards">
+      <div class="card">
+        <div class="card-image sphere">
+          <img src="images/sphere.svg" class="sp-image" alt="">
+        </div>
+        <h2 class="card-text2">Languages</h2>
+        <ul class="alist">
+          <li><a href="#" class="anc2">Javascript</a></li>
+          <li><a href="#" class="anc2">Ruby</a></li>
+          <li><a href="#" class="anc2">Html</a></li>
+          <li><a href="#" class="anc2">CSS</a></li>
+        </ul>
+
```

Comment:
- [ ] Kindly stick to the Figma template as much as possible for a better experience. Currently, on the `mobile version` of your project, the `Languages`, `Frameworks`, and `Skills` cards do not look like what we have in the Figma template you worked on. Kindly fix that and let them look alike.

- Also please attach a `border` and `border radius` around the `Ruby on rails`, `CSS`, `JavaScript`, and `HTML` as seen in the Figma design for the mobile version.

Figma Design 👇 

![mob rel](https://user-images.githubusercontent.com/78634841/206762704-415a9a69-de5f-44ea-8122-6eee900e5969.png)

Your design 👇 

![dan mob](https://user-images.githubusercontent.com/78634841/206762783-1eda488c-b185-4636-8ad4-ecde6b912481.png)